### PR TITLE
Avoid panic caused by deleting git folders

### PR DESCRIPTION
**Pull request by jm-observer**
the reason of https://github.com/cargo-generate/cargo-generate/issues/619  is to delete _.git_ folder, so  instead of deleting _.git_, I don't copy _.git_

### CODE REVIEW

File: src/git/utils.rs [hunk 1]
```diff
@@ -43,7 +43,7 @@ pub fn clone_git_template_into_temp(
     let branch = get_branch_name_repo(&repo)?;
 
     // change from git repository into normal folder.
-    remove_history(git_clone_dir.path())?;
+    // remove_history(git_clone_dir.path())?;
```
Comment (sassman) [inline src/git/utils.rs:46]: could you please remove this code here, and also the now unused function?

**Review by sassman — CHANGES_REQUESTED** [review]
Thanks for the neat fix, this is indeed way better, the less is copied the less needs to be deleted :)

File: src/git/utils.rs (hunk 1 above)
Comment (jm-observer) [reply src/git/utils.rs]: done!

**Review by sassman — APPROVED** [review]
Thanks

**Pull request merged**